Source/UIKit/CollectionViewDataSource.swift

```diff
@@ -52,6 +52,9 @@ extension UICollectionView: DataSourceView {
     
     
     public func dequeueCell(reuseIdentifier: String, indexPath: IndexPath) -> UICollectionViewCell {
+        layer.drawsAsynchronously = true
+        layer.shouldRasterize = true
+        layer.rasterizationScale = UIScreen.main.scale
```

Comment:
Hmm, I don’t think this is the right place for those property adjustments — this’ll be hit every time a cell is dequeued when really they should only need to be called once.